bgp/modules/terms.py

```diff
@@ -238,14 +238,25 @@ class IsbnExtractorModule(ExtractorModule):
     @staticmethod
     def validate_isbn(isbn):
         isbn = isbn.replace("-", "").replace(" ", "").upper();
-        match = re.search(r'^(\d{9})(\d|X)$', isbn)
-        if not match:
+        match10 = re.search(r'^(\d{9})(\d|X)$', isbn)
+        match13 = re.search(r'^(\d{12})(\d)$', isbn)
+
+        if match10:
+            digits = match10.group(1)
+            check_digit = 10 if match10.group(2) == 'X' else int(match10.group(2))
+            result = sum((i + 1) * int(digit) for i, digit in enumerate(digits))
+            if (result % 11) == check_digit:
+                return match10.group()
+        elif match13:
+            digits = match13.group()
+            if len(digits) != 13:
+                return False
+            product = (sum(int(ch) for ch in digits[::2])
+                       + sum(int(ch) * 3 for ch in digits[1::2]))
+            if product % 10 == 0:
+                return match13.group()
```

Comment:
Look into replacing all of this using [isbnlib](https://pypi.org/project/isbnlib/) to simplify.

Reply:
Sounds good, we can open that as a new issue